template/task-templates.wdl

```diff
@@ -69,29 +63,23 @@ task dynamic_disk_and_ram_task {
         memory: "~{memory_gb} GB"
         disk: "~{disk_size_gb} GB"
         container: ""
-        maxRetries: max_retries
+        maxRetries: 1
     }
 }
 
 task use_all_cores_task {
     meta {
-        description: "This template is appropriate for all tasks which can be run on multiple cores. Update the default disk and RAM allocations, or copy and paste from `dynamic_disk_and_ram_task` as appropriate." 
+        description: "This template is appropriate for all tasks which can be run on multiple cores. Update the default disk and RAM allocations, or copy and paste from `dynamic_disk_and_ram_task` as appropriate."
     }
 
     parameter_meta {
-        use_all_cores: "Use all cores? Recommended for cloud environments. Not recommended for cluster environments."
+        use_all_cores: "Use all cores? Recommended for cloud environments."
         ncpu: "Number of cores to allocate for task"
-        memory_gb: "RAM to allocate for task, specified in GB"
-        disk_size_gb: "Disk space to allocate for task, specified in GB"
-        max_retries: "Number of times to retry in case of failure"
     }
 
     input {
         Boolean use_all_cores = false
-        Int ncpu = 1
-        Int memory_gb = <>
-        Int disk_size_gb = <>
-        Int max_retries = 1
+        Int ncpu = <>
```

Comment:
I suppose this comes down to how one thinks about templates, but should we leave a reasonable default, e.g. `Int ncpu = 1`. I would think it would be clear that one might want to update this, but it would have a valid value. The same may be true for `memory` and `disk` in the `runtime` section.

Reply:
I think a reasonable setting for all of them would be our lower bound for the repo (which should also be documented). So 4gb mem, 10gb disk, 2cpu. I'll go and make those updates.

Reply:
I think setting these argues in favor of providing a default for `container` as well, but I'm interested to hear your opinion as well.